nautobot_chatops/templates/nautobot_chatops/chatops_account_link_edit.html

```diff
@@ -0,0 +1,68 @@
+{% extends 'generic/object_edit.html' %}
+{% load static %}
+{% load form_helpers %}
+
+{% block form %}
+<div class="panel panel-default">
+    <div class="panel-body">
+        {% render_field form.email %}
+        {% render_field form.platform %}
+        <div class="form-group">
+            <label class="col-md-3 control-label"></label>
+            <div class="col-md-9 pull-right">
+                <button type="button" class="btn btn-primary" id="user-id-lookup">Look up User ID from Email</button>
+                <img id="lookup-loader" src="{% static 'img/ajax-loader.gif' %}" height="20" style="display: none;">
+            </div>
+        </div>
+        {% render_field form.user_id %}
+    </div>
+</div>
+{% endblock %}
+
+{% block javascript %}
+<script type="text/javascript">
+$(document).ready(function() {
+    $("#user-id-lookup").click(function() {
+        let platform = $("#id_platform").val();
+        if (!platform) {
+            alert("Please select a platform");
+            return;
+        }
+        let email = $("#id_email").val();
+        if (!email) {
+            alert("Please enter an email to look up the value for");
```

Comment:
Im not sure how to make this sound better, perhaps:
```suggestion
            alert("Please enter an email to look up your User ID");
```